misocho/questioner
# #162970628 Adds User Homepage

#### What does this PR do?
Creates the user homepage index.html
#### Description of Task to be completed?
* Create user homepage
#### How should this be manually tested?
* Clone the repo
* Navigate to the project directory
* Check out the branch for this PR  ft-user-homepage-162970628
* Open index.html with your browser
#### What are the relevant pivotal tracker stories?
#162970628
#### Screenshots
![screenshot_1](https://user-images.githubusercontent.com/29132159/50725306-da399f80-110c-11e9-9749-dab90c1dda63.png)
